spec/integ/matrix-client-room-timeline.spec.js

```diff
@@ -252,11 +252,14 @@ describe("MatrixClient room timelines", function() {
                 client.scrollback(room).done(function() {
                     expect(room.timeline.length).toEqual(1);
                     expect(room.oldState.paginationToken).toBe(null);
-                    done();
+
+                    // still have a sync to flush
+                    httpBackend.flush("/sync", 1).then(() => {
```

Comment:
I'm not sure the `afterEach` should be used for that, should it not just kill the httpBackend altogether?